docs/camino-messenger/bot/installation.mdx

```diff
@@ -32,6 +32,47 @@ software](#compiling-from-source) from its source code, while the second
 method [utilizes Docker](#building-docker-image) to create an image and run a
 container.
 
+## Known Issues
+
+The Camino Messenger Bot the following known issues should be considered:
+
+**Version: 10.2.0**
+
+- After start-up of a fresh bot installation takes 10 to 15 seconds before messages
+  can be sent or received. This period can be more depending on the network 
+  connections and if the bot receives messages before those 15 seconds.
+
```

Comment:
Is this only happens when there is a _fresh installation_? I was under impression that this happens when the bot is starting and almost everytime with some of the partners? (Never happened with me)